pandora-common/package.json

```
@@ -49,6 +49,7 @@
 		"@typescript-eslint/eslint-plugin": "6.18.1",
 		"@typescript-eslint/parser": "6.18.1",
 		"eslint": "8.56.0",
+		"eslint-plugin-deprecation": "^2.0.0",
```

Please use exact version for dependency:
```suggestion
		"eslint-plugin-deprecation": "2.0.0",
```
And then run `pnpm i` and commit both this change and updated lockfile.